main.cpp

```diff
@@ -64,14 +64,20 @@ send me a DM to check your pull request
 #include <string>
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
-    //2
-    //3
+
+    int value;
+    std::string name;
+
+    T(int v, const char* s)   //1
+    {
+        value = v; //2
+        name = s; //3
+    }
```

Comment:
Use the member initializer list to init your class members with the parameters.
The reason: you are initializing them with their default values when they're constructed.
Then you're initializing an instance on the right side of the `=`, which must then invoke the `operator=` to assign them.  That's a double initialization. 
- [ ] revise

Reply:
done